src/sections/quiz/quiz-header.js

```diff
@@ -0,0 +1,38 @@
+import Button from "@mui/material/Button";
+import {SvgIcon} from "@mui/material";
+import Container from "@mui/material/Container";
+import Box from "@mui/material/Box";
+import Logo from "../../components/svg-color/Logo.svg";
+import {Lora} from '@next/font/google';
+import { useTheme } from '@mui/material/styles';
+
+
+const lora = Lora({preload: false});
+
+export default function QuizHeader() {
+  const theme = useTheme()
+  return (
+    <Box
+      component="header"
+      sx={{
+        backgroundColor: theme.palette.background.quiz,
+        padding: '1.7em',
+        alignItems: 'center',
+      }}
+    >
+      <Container maxWidth="lg" sx={{
+        display: "flex",
+        justifyContent: "space-between",
```

Comment:
The same staff, we have a single element inside, no need to align it with flex and justifyContent